pittapi/laundry.py

```diff
@@ -37,7 +37,22 @@
 }
 
 
-def _get_laundry_info(building_name: str) -> Any:
+class BuildingStatus(NamedTuple):
```

Comment:
Why is this class inheriting `typing.NamedTuple`? Did you mean `collections.NamedTuple`?
I think all you need/want is a `dataclass`. Also, how does this affect downstream serialization? That is, can I still dump this to a valid JSON object?

Reply:
Yes, I meant to use `typing.NamedTuple` and not `collections.namedtuple`:
1. `typing.NamedTuple` has (imo) a cleaner syntax than `collections.namedtuple`.
2. `typing.NamedTuple` is typed, while `collections.namedtuple` isn't. One of my motivations for making this PR is to gradually prep the codebase so that we can introduce `mypy` to the repo, and having explicitly defined types for data fields will help with that.
3. The PittAPI repo had already been using `typing.NamedTuple` long before I started contributing to it, so I'm also sticking to it for consistency.

`dataclass` would work for our purposes as well, and I don't think there's any effect on serialization in either case (both `dataclass` and `typing.NamedTuple` can be converted to plain Python `dict`, which could then be serialized to JSON). I don't know if `dataclass` provides any features/benefits that'd make it notably better than `typing.NamedTuple`, at least for our purposes.